pkg/common/bitmap/bitmap.go

```diff
@@ -47,11 +76,51 @@ func (n *Bitmap) Iterator() Iterator {
 	return &BitmapIterator{i: 0, bm: n}
 }
 
+func wordnum(i uint64) uint64 {
+	return i / bits_per_bitmapword
```

Comment:
I don't know if golang compiler actually will turn this into shift and mask, but if we are doing bits, you should shift this.   Actually easier to understand in the context of bit mask.